packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -467,7 +467,7 @@ export class MonacoEditorProvider {
 
                 if (referencesController._widget) {
                     referencesController._widget.show(range);
-                    referencesController._widget.focus();
+                    referencesController._widget.focusOnReferenceTree();
```

Comment:
wonder whether we still need `installReferencesController`

Reply:
At previous update I investigated this area and found that we keep `referencesController` because `Theia` creates `standaloneCodeEditor` instance every time an editor is open https://github.com/eclipse-theia/theia/pull/7149#discussion_r390786470

I checked and can confirm that [the comment](https://github.com/eclipse-theia/theia/pull/7149#discussion_r390786470) still relevant, so we need keep `referencesController`